Use distro nlohmann_json; ABF has no network for FetchContent git clone.

--- a/CMakeLists.txt	2026-07-22 02:04:20.000000000 +0200
+++ b/CMakeLists.txt	2026-09-07 00:51:09.695864422 +0200
@@ -281,12 +281,7 @@
     endif()
 endif()
 
-include(FetchContent)
-FetchContent_Declare(json
-    GIT_REPOSITORY https://github.com/nlohmann/json.git
-    GIT_TAG v3.12.0
-)
-FetchContent_MakeAvailable(json)
+find_package(nlohmann_json REQUIRED)
 
 if("${INSTALL_PREFIX}" STREQUAL "")
     set(INSTALL_PREFIX "./")
